utils/bcm27xx-eeprom/Makefile

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2020 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=bcm27xx-eeprom
+PKG_VERSION:=a9ca308223c1d0426b9ab320696b95954078c3b4
```

Comment:
Is the current version of the Raspbian rpi-eeprom package also based on this version? (I don't have a Pi available right now to check.)

If it isn't, would prefer if this package tracked the Raspbian package (which would be the closest thing to a "release" version). (If it has to be a newer version, would appreciate a reason why.)

Reply:
I don't know about Raspbian, but rpi-eeprom is continuously evolving and distributions like libreelec are updating to the latest version available, so I think that using the latest version available is the way to go.

Reply:
Always updating to the latest version available sounds dangerous for a package that can cause the device to not boot properly.

Reply:
I suggest keeping this version and try to minimise the updates of this package.